### ISSUE TITLE
Open the pages inside the Notion Windows app

### ISSUE THREAD

**Issue by Shazzik** [danielduckworth/Flow.Launcher.Plugin.NotionSearch#4]
Hello! Thanks a lot for this brilliant plugin 👌

I was wondering if it was possible to open pages in the Windows app instead of a browser? Perhaps [this ](https://thomasjfrank.com/how-to-share-notion-links-that-open-directly-in-the-app/)might help?

**Comment by danielduckworth** [comment]
Hi Shazzik, that's a great idea. I'll do some testing and figure out a good way to use integrate desktop app links.

**Comment by branislavbudzak** [comment]
I would also love to be able to open Notion pages in Windows App. Do you figure it out yet?

**Comment by danielduckworth** [comment]
> I would also love to be able to open Notion pages in Windows App. Do you figure it out yet?

Yeah it looks like the links just need to use a notion prefix instead of http. I think the safest way to implement this initially would be to add a user setting toggle to make links open in the app as the default.

Then it can be improved by adding context options to open in the browser or open in the app.

**Comment by branislavbudzak** [comment]
> > I would also love to be able to open Notion pages in Windows App. Do you figure it out yet?
> 
> Yeah it looks like the links just need to use a notion prefix instead of http. I think the safest way to implement this initially would be to add a user setting toggle to make links open in the app as the default.
> 
> Then it can be improved by adding context options to open in the browser or open in the app.
Yeah, that sounds great. Hope you will find time to implement it.

**Comment by arizzubair** [comment]
To make it even easier to implement, you can give user an option of the url. User can input `"notion://www.notion.so/"` for opening results in app. And, the default could still be left as the http url. 

Here's the alfred plugin's relevant code: https://github.com/wrjlewis/notion-search-alfred-workflow/blob/master/notion.py#L109
